### ISSUE TITLE
Don't let users drag and drop embedded clauses

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/web-components#178]
# Feature Request 🛍️
Make it clear that users can not drag and drop embedded clauses. Embedded clauses currently have the same drag and drop UI as normal smart clause templates, implying that they can be dragged and dropped.

## Possible Solution
Remove drag and drop UI from embedded Clauses and give some padding so the borders aren't intersecting with the parent clause

This examples is in the copyright license Clause
![image](https://user-images.githubusercontent.com/40569793/91341300-4d42ce80-e7a7-11ea-958b-28a3de4ac929.png)

**Comment by Michael-Grover** [comment]
Users probably shouldn't be able to delete embedded clauses either, right?

**Comment by jeromesimeon** [comment]
> Users probably shouldn't be able to delete embedded clauses either, right?

They can do it in contract templates like this: https://templates.accordproject.org/copyright-license@0.15.0.html

<img width="1068" alt="Screenshot 2020-08-26 at 2 32 41 PM" src="https://user-images.githubusercontent.com/670099/91342451-0ce45000-e7a9-11ea-84dc-3f143a807bfd.png">

I think the plan is to clarify contract v clause templates down the road, including in the editor.

**Comment by sanketshevkar** [comment]
@Michael-Grover I'll like to work on this feature.

**Comment by Michael-Grover** [comment]
@sanketshevkar I've assigned you, thanks!

**Comment by sanketshevkar** [comment]
![bug](https://user-images.githubusercontent.com/43399522/107730203-d380ac00-6d18-11eb-8bc5-8832c9282c83.png)
This seems like an embedded clause but it does not have the 'Embedded 'title and shares the same drag and drop Icon with the main clause.

**Comment by irmerk** [comment]
Yes @sanketshevkar I don't think they will have "Embedded" in the title.
